README.md

```diff
@@ -101,6 +101,11 @@ Be aware that Cloud Shell has [several limitations][cloud-shell-limitations],
 in particular an inactivity timeout that will close running shells after 20
 minutes. Please consider it only for blueprints that are quickly deployed.
 
+## VM Image Support
+
+The HPC Toolkit supports the HPC CentOS 7 VM image and the Ubuntu 20.04 LTS
```

Comment:
nit: I would add the word `officially` in here. I have used other images with the Toolkit with some success. Optionally, make some statement unsupported images, that they might work but are not actively tested.

Reply:
Added both.

Reply:
Still nit: My lingering concern is that someone will read this section and be like, "Well, if they don't support Rocky then I guess I am not interested".

I think the section you added in the images doc is great. I am wondering if we can tie it in somehow here to keep from losing these people. 

Maybe saying:

>The HPC Toolkit **officially** supports the following VM images:
>...
>
>For more information **on these and other images**, see [docs/vm-images.md](docs/vm-images.md).

(bolding used to show additions)

Reply:
I see, I was looking at the wrong file when I updated. It should be updated now.